### ISSUE TITLE
kernel-memory keyword exaction handler is using hard-code prompt instead of reading from prompt file

### ISSUE THREAD

**Issue by vtjc2002** [microsoft/Document-Knowledge-Mining-Solution-Accelerator#22]
Unlike the kernel-memory/service/Core/Handlers/SummarizationHandler.cs, kernel-memory/service/Core/Handlers/KeywordExtractingHandler.cs doesn't use kernel-memory/service/Core/Prompts/extract-keywords.txt to allow extrat keyword prompt to be configured without changing the code.

Which contradicts the documentation in Data_Processing.md

Please modify so the KeywordExtractingHandler reads the prompt from extract-keywords.txt

**Comment by Vinay-Microsoft** [comment]
@vtjc2002 Thanks for your feedback. Will pass it on to our engineering team. Will keep you posted.

**Comment by vtjc2002** [comment]
Hi, I created PR for the fix.  I followed the pattern defined by the SummarizationHandler.cs

**Comment by Prasanjeet-Microsoft** [comment]
> Hi, I created PR for the fix. I followed the pattern defined by the SummarizationHandler.cs

Hello @vtjc2002, Thank you for highlighting this and providing the code fix. We will review your code with the Engineering team and provide an update accordingly.

Thank you.

**Comment by Prasanjeet-Microsoft** [comment]
Hello @vtjc2002, I just wanted to update you that we are still awaiting code review and approval from the Engineering team. Once we receive that, we will provide an update accordingly.

Thank you.

**Comment by Prasanjeet-Microsoft** [comment]
Hello @vtjc2002, I just wanted to update you that we are currently reviewing the code and performing validation. Once that is complete, we will approve your changes and provide you with an update as soon as possible.

Thank you.

**Comment by Prasanjeet-Microsoft** [comment]
Hello @vtjc2002, I just wanted to update you that we have completed the code review, performed the validation and approved your changes.

Thank you.

**Comment by Prasanjeet-Microsoft** [comment]
Hello @vtjc2002, Since the PR has been approved and merged into the main branch, can we go ahead and close this GitHub issue?

**Comment by Prasanjeet-Microsoft** [comment]
Hello @vtjc2002, 

> Since the PR has been approved and merged into the main branch, can we go ahead and close this GitHub issue?

Hello @vtjc2002, Since the PR has been approved and merged into the main branch, should we go ahead and close this GitHub issue now?
